### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by yashnaravade**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (surajshende247) [inline .vscode/settings.json:1]: don't push this file, add to .gitignore

File: client/package.json [hunk 2]
```diff
@@ -7,8 +7,10 @@
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.1.2",
-    "bootstrap": "^5.1.3",
+    "bootstrap": "^5.2.2",
```
Comment (surajshende247) [inline client/package.json:10]: not required

File: client/package.json [hunk 3]
```diff
@@ -7,8 +7,10 @@
...
+    "font-awesome": "^4.7.0",
     "react": "^18.0.0",
+    "react-bootstrap": "^2.5.0",
```
Comment (surajshende247) [inline client/package.json:13]: not required

File: client/src/components/Footer/Footer.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
+*, *:after, *:before {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    }
+
+    
+a{
```
Comment (surajshende247) [inline client/src/components/Footer/Footer.css:8]: indentation

File: client/src/components/Footer/Footer.css [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+    content: "";
+    height: 1px;
+    margin: 0 auto;
+    position: absolute;
+    right: 80px;
+    text-align: center;
+    top: 1px;
+    width: 100px;
+    }
+
```
Comment (surajshende247) [inline client/src/components/Footer/Footer.css:53]: remove this whitespace

File: client/src/components/Footer/Footer.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './Footer.css'
+
+function Footer() {
+  return (
+    <footer>
```
Comment (surajshende247) [inline client/src/components/Footer/Footer.js:6]: indentation

File: client/src/components/Home/Home.css [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+  .form-row > .col,
+  .form-row > [class*="col-"] {
+    padding-right: 5px;
+    padding-left: 5px;
+  }
+
+  .bmi-calculator{
+    background-color: var(--dark);
+  }
+
```
Comment (surajshende247) [inline client/src/components/Home/Home.css:126]: remove these empty line and check indentation

**Review by surajshende247 — CHANGES_REQUESTED** [review]
Use images from `storyset`

File: client/package.json (hunk 3 above)
Comment (yashnaravade) [reply client/package.json:13]: It is required for the carousel. Not working with the default bootstrap

File: client/src/components/Footer/Footer.css (hunk 4 above)
Comment (yashnaravade) [reply client/src/components/Footer/Footer.css:8]: okay

**Review by surajshende247 — APPROVED** [review]

**Pull request merged**